docs/cypress-testing-library/intro.md

```diff
@@ -53,9 +52,12 @@ To show some simple examples (from
 
 ```javascript
 cy.findAllByText('Jackie Chan').click()
-cy.queryByText('Button Text').should('exist')
-cy.queryByText('Non-existing Button Text').should('not.exist')
-cy.queryByLabelText('Label text', {timeout: 7000}).should('exist')
+cy.findByText('Button Text').should('exist')
+cy.findByText('Non-existing Button Text').should('not.exist')
+cy.findByLabelText('Label text', {timeout: 7000}).should('exist')
+
+// findAllByText _inside_ a form element
+cy.get('form').findByText('Button Text').should('exist')
 cy.get('form').within(() => {
```

Comment:
Maybe we should remove this example in favor of the regular chaining one?

Reply:
I think so. I avoid `cy.within` for many of the same reasons I avoid `with` in JavaScript. At the top level and in the simplest case, it avoids duplicate code, but it can easily get out of hand

Reply:
Cool. Let's remove it then 👍